.github/workflows/main.yml

```diff
@@ -12,19 +12,18 @@ jobs:
 
     - uses: actions/setup-ruby@v1
       with:
-        version: '2.6'
+        version: '2.7'
```

Comment:
As a nice to have, using [`ruby/setup-ruby`](https://github.com/ruby/setup-ruby) means we wouldn't need to specify a version here.

Reply:
Ah, this has already been merged in. I was too slow! I'll raise another pull request to do this.